internal/faustjs.org/docs/blocks/gutenberg-blocks.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: gutenberg-blocks-provider
+title: Getting Started with Gutenberg Blocks Provider
+---
+
+# Getting Started with Gutenberg Blocks Provider
+
+## Quick Start
+
+:::note
+
+Make sure you have completed the initial setup for Faust at [Getting Started](/docs/getting-started).
+
+:::
+
+Install the blocks:
+
+```bash
+npm i @faustwp/blocks
+```
+
+Open ```_app.js``` and import the blocks provider:
+
+```js
+import { WordPressBlocksProvider } from '@faustwp/blocks';
+```
```

Comment:
There is a missing step to setup the `WordPressBlocksProvider` here in `_app.js`